core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -20,15 +20,9 @@
  *
  * @author Leo Feyer <https://github.com/leofeyer>
  */
+#[AllowDynamicProperties]
```

Comment:
```suggestion
 * @property Template $Template Will be made protected in Contao 5.0
 *
 * @author Leo Feyer <https://github.com/leofeyer>
 */
#[AllowDynamicProperties]
```

Should we add an info that this will change in Contao 5.0?

Reply:
Since there is nothing to do in this file (the `$Template` property will be added to the `Controller` class), I did not add a `@todo`. But we might as well add one.

Reply:
> the `$Template` property will be added to the `Controller` class

We sure about that? _(But I guess I should proably create a separate issue for that...)_

Reply:
Yes, I guess:

https://github.com/contao/contao/blob/dba2cf227e062bfc06e6b8c1a2c52c8df4fbb21b/core-bundle/src/Resources/contao/library/Contao/Controller.php#L52-L57

But of course we can discuss this. (Preferably in a separate issue.)